content/soarian/r4/clinical/summary/allergy-intolerance.md

```diff
@@ -0,0 +1,130 @@
+---
+title: AllergyIntolerance | R4 API
+---
+
+# AllergyIntolerance
+
+* TOC
+{:toc}
+
+## Overview
+
+
+The AllergyIntolerance resource provides the clinical assessment of a patient’s allergy when exposed to a specific substance or class of substance, including information about the adverse reaction. _Soarian Clinicals_<sup>®</sup> does not differentiate between an allergy and an intolerance. The term __allergy__ is used generically and may overlap with an __intolerance__. Substances include medications, foods, and environmental allergens (such as plants and animals). 
```

Comment:
Remove italics from Soarian Clinicals.

Reply:
In the Overview section, "Soarian Clinicals" still has italics. Please remove this markdown.

PRODOC-160857.

Reply:
Done